src/simulators/statevector/statevector_state.hpp

```diff
@@ -333,54 +333,54 @@ class State : public QuantumState::StateChunk<statevec_t> {
-     {"u1", Gates::mcp},  // zero-X90 pulse waltz gate
-     {"u2", Gates::mcu2}, // single-X90 pulse waltz gate
-     {"u3", Gates::mcu3}, // two X90 pulse waltz gate
-     {"u", Gates::mcu3},  // two X90 pulse waltz gate
-     {"U", Gates::mcu3},  // two X90 pulse waltz gate
-     /* 2-qubit gates */
-     {"CX", Gates::mcx},       // Controlled-X gate (CNOT)
-     {"cx", Gates::mcx},       // Controlled-X gate (CNOT)
-     {"cy", Gates::mcy},       // Controlled-Y gate
-     {"cz", Gates::mcz},       // Controlled-Z gate
-     {"cp", Gates::mcp},       // Controlled-Phase gate
-     {"cu1", Gates::mcp},      // Controlled-u1 gate
-     {"cu2", Gates::mcu2},     // Controlled-u2 gate
-     {"cu3", Gates::mcu3},     // Controlled-u3 gate
-     {"cu", Gates::mcu},       // Controlled-u4 gate
-     {"cp", Gates::mcp},       // Controlled-Phase gate
-     {"swap", Gates::mcswap},  // SWAP gate
-     {"rxx", Gates::rxx},      // Pauli-XX rotation gate
-     {"ryy", Gates::ryy},      // Pauli-YY rotation gate
-     {"rzz", Gates::rzz},      // Pauli-ZZ rotation gate
-     {"rzx", Gates::rzx},      // Pauli-ZX rotation gate
-     {"csx", Gates::mcsx},     // Controlled-Sqrt(X) gate
-     {"csxdg", Gates::mcsxdg}, // Controlled-Sqrt(X)dg gate
-     {"ecr", Gates::ecr},      // ECR Gate
-     /* 3-qubit gates */
-     {"ccx", Gates::mcx},      // Controlled-CX gate (Toffoli)
-     {"cswap", Gates::mcswap}, // Controlled SWAP gate (Fredkin)
-     /* Multi-qubit controlled gates */
+    {                           // 1-qubit gates
+     {"delay", Gates::id},      // Delay gate
+     {"id", Gates::id},         // Pauli-Identity gate
+     {"x", Gates::mcx},         // Pauli-X gate
+     {"y", Gates::mcy},         // Pauli-Y gate
+     {"z", Gates::mcz},         // Pauli-Z gate
+     {"s", Gates::s},           // Phase gate (aka sqrt(Z) gate)
+     {"sdg", Gates::sdg},       // Conjugate-transpose of Phase gate
+     {"h", Gates::h},           // Hadamard gate (X + Z / sqrt(2))
+     {"t", Gates::t},           // T-gate (sqrt(S))
+     {"tdg", Gates::tdg},       // Conjguate-transpose of T gate
+     {"p", Gates::mcp},         // Parameterized phase gate
+     {"sx", Gates::mcsx},       // Sqrt(X) gate
+     {"sxdg", Gates::mcsxdg},   // Inverse Sqrt(X) gate
+                                // 1-qubit rotation Gates
+     {"r", Gates::mcr},         // R rotation gate
+     {"rx", Gates::mcrx},       // Pauli-X rotation gate
+     {"ry", Gates::mcry},       // Pauli-Y rotation gate
+     {"rz", Gates::mcrz},       // Pauli-Z rotation gate
+                                // Waltz Gates
+     {"u1", Gates::mcp},        // zero-X90 pulse waltz gate
+     {"u2", Gates::mcu2},       // single-X90 pulse waltz gate
+     {"u3", Gates::mcu3},       // two X90 pulse waltz gate
+     {"u", Gates::mcu3},        // two X90 pulse waltz gate
+     {"U", Gates::mcu3},        // two X90 pulse waltz gate
+                                // 2-qubit gates
```

Comment:
Here's some more unrelated formatting changes, but these ones have a problem too: some of the semantic meaning from the comment positioning is getting lost.